docs/content-services/components/content-node-selector-panel.component.md

```diff
@@ -45,6 +45,7 @@ Opens a [Content Node Selector](content-node-selector.component.md)  in its own
 | showDropdownSiteList | `boolean` |  | Toggle sites list dropdown rendering |
 | showFilesInResult | `void` |  | Shows the files and folders in the search result |
 | showSearch | `boolean` |  | Toggle search input rendering |
+| showNodeCounter | `boolean` | false | Shows the node counter in the breadcrumb |
```

Comment:
do we need an option to enable or disable? or we can have it all the time?

Reply:
We do. As the issue specifies, it should not be shown when the action is moving or copying.

Reply:
why we don't want in that case?

Reply:
I'm assuming because there is always a single node selected when moving/copying. Either the folder selected or the parent folder. Therefore making the counter redundant as it always displays "1". Maybe @iuliaib could give a better answer.

Reply:
The reason is the one explained by Thomas.
@eromano please check the current behaviour in the description of the attached [AAE-5124](https://alfresco.atlassian.net/browse/AAE-5124) ticket for a better understanding.
 It doesn't make any sense to see always "1 selected" even if you didn't select a file or folder from Destination Picker. Every time you click on Copy/Move, you see "1 selected" by default.

Reply:
Agree I don't any valid reason why we should have the selection in other places.

Reply:
alright thanks It was quite obvious, I just wanted to check if everyone was paying attention 🥇  :)